docs/source/nodeoperations/restore.md

```diff
@@ -149,6 +160,32 @@ $ kubectl wait --for='condition=Available=True' -n scylla scyllaclusters.scylla.
 scyllacluster.scylla.scylladb.com/target condition met
 ```
 
+:::{caution}
+### Restoring schema with **ScyllaDB OS 5.4.X** or **ScyllaDB Enterprise 2024.1.X** and `consistent_cluster_management`
+
+After you've followed the above steps with a ScyllaDB target cluster with `consistent_cluster_management` disabled, you'll need to enable Raft by configuring the target cluster with `consistent_cluster_management: true`.
+Refer to [API Reference](../api-reference/index.rst) to learn how to customize ScyllaDB configuration files.
+
+You will then need to execute a rolling restart of the ScyllaCluster for the change to take effect.
+```console
+kubectl patch scyllacluster target --type merge -p '{"spec": {"forceRedeploymentReason": "raft enabled"}}'
+```
+
+Use the following commands to wait until restart is finished:
+```console
+$ kubectl wait --for='condition=Progressing=False' -n scylla scyllaclusters.scylla.scylladb.com/target
```

Comment:
```suggestion
$ kubectl wait --for='condition=Progressing=False' scyllaclusters.scylla.scylladb.com/target
```
you don't have the namespace in the previous patch command either nor you should, the namespace is user chosen and there are more then one so this prevents copy pasting the commands

Reply:
applies globally

Reply:
I disagree it prevents copy pasting, you have to adjust the namespace either way unless you use the default. And in this case the example manifests were specifically setting the scylla namespace. I don't want to go back and forth on this though, so applied.

Reply:
> unless you use the default

you **can** change the working namespace and that's what people mostly do. some options to do that:
```
alias kubens='kubectl config set-context --current --namespace'
```
```
oc project <ns>
```
and some plugins...

Reply:
Also removed the namespace from metadata of SC manifests